main.go

```diff
@@ -137,7 +137,8 @@ func (c *component) New(ctx context.Context, conf *config) (func(context.Context
 		"insert":                     serverfull.NewFunction(insert.Handle),
 		"fetchByIP":                  serverfull.NewFunction(fetchByIP.Handle),
 		"fetchByHostname":            serverfull.NewFunction(fetchByHostname.Handle),
-		"fetchByArnID":               serverfull.NewFunction(fetchByArnID.Handle),
+		"fetchByArnID":               serverfull.NewFunction(fetchByResourceID.Handle),
```

Comment:
👍 , we will be able to remove ByArnID later once we are sure the client is updated.